### PR TITLE
Fix issues in the release note

### PR DESCRIPTION
**Pull request by MaryamZi**
## Purpose
$title.

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-alpha1/RELEASE_NOTE.md [hunk 1]
```diff
@@ -402,7 +404,7 @@ The rest descriptor type of the `InclusiveCongifuration` is `anydata` and that o
 
 ##### Improved Listener Declaration  
 
-The listener declaration is improved to accept listener values that are a subtype of both the compiler built-in type `listener` and `error` type. This allows the following listener declaration when the `init` method of `Listener` may return an error.
+The listener declaration is improved to accept listener types that may return an error on initialization. This allows the following listener declaration where the `init` method of `Listener` may return an error.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-alpha1/RELEASE_NOTE.md:407]: @rdhananjaya, please check. 

A value cannot be a subtype of both the listener and error, right? Also the listener declaration doesn't quite accept the value when it is an error. It is only the type that is allowed?

Comment (rdhananjaya) [reply downloads/swan-lake-release-notes/swan-lake-alpha1/RELEASE_NOTE.md:407]: +1 for the change, but in `listener T l = expr`, `expr` is a value, not a type.

Comment (rdhananjaya) [reply downloads/swan-lake-release-notes/swan-lake-alpha1/RELEASE_NOTE.md:407]: May be we should say `listener class that may return an error ...`

Comment (MaryamZi) [reply downloads/swan-lake-release-notes/swan-lake-alpha1/RELEASE_NOTE.md:407]: Yeah I meant using `T` in the listener declaration when initialization of `T` may return an error.

```ballerina
listener T l1 = new;
listener l2 = new T();
```

+1 for using class. 

How about
```suggestion
The listener declaration is improved to allow using listener classes that may return an error on initialization. This allows the following listener declaration where the `init` method of `Listener` may return an error.
```

Comment (rdhananjaya) [reply downloads/swan-lake-release-notes/swan-lake-alpha1/RELEASE_NOTE.md:407]: +1

**Review by rdhananjaya — APPROVED** [review]

**Review by praneesha — APPROVED** [review]

**Pull request merged**